### DISCUSSION TITLE
RuntimeError: optional has no value

### DISCUSSION THREAD

**Discussion by Carinazhao22** [jax-ml/jax#6244] [Q&A] [answered]
Hi, I got 'RuntimeError: optional has no value' when executing 'key=random.PRNGKey(0)'. How can i fix the issue? Thanks
version: jaxlib==0.1.62+cuda112
 
![image0](https://user-images.githubusercontent.com/52422263/112691248-43b33f00-8e3a-11eb-92e3-24435aa15caa.jpg)

**Comment by mattjj** [comment, marked as answer]
Thanks for the report!

Hmm, I don't think we've seen this one before, and it's a pretty low-level error. I think the best first step is to update your jaxlib installation, from 0.1.62 to the pypi-current 0.1.64. I think this might work:

```
pip install --upgrade jaxlib==0.1.64+cuda112 -f https://storage.googleapis.com/jax-releases/jax_releases.html
```

**Reply by Carinazhao22** [reply]
thanks, it works.

**Comment by jlevy44** [comment]
I seem to be getting this error as well! Will try the upgrade.

**Comment by jlevy44** [comment]
Seemed to work for me!

**Reply by mattjj** [reply]
That's great, thanks for letting us know!
